.changeset/beige-dodos-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@onflow/transport-http": patch
```

Comment:
TODO: because of recently pinned dependency versions, there's some issues with downstream version bumps.  I'm trying to figure out a good way to solve this before merging.